jenkins/scripts/integration_test.sh

```diff
@@ -74,8 +74,12 @@ elif [[ "${GINKGO_FOCUS}" == "pivoting" ]] || [[ "${GINKGO_FOCUS}" == "remediati
 fi
 echo "Running in region: ${OS_REGION_NAME}"
 
-if [[ "${TESTS_FOR}" == "e2e_tests"* && ( "${GINKGO_FOCUS}" != "integration" && "${GINKGO_FOCUS}" != "basic" ) ]]; then
+if [[ "${GINKGO_FOCUS}" == "pivoting" ]] || [[ "${GINKGO_FOCUS}" == "remediation" ]] ||
```

Comment:
This works, but I think at this point we should use `case` to make it easier to maintain and read. Let's do that next time we touch this.

Reply:
good suggestion. Next time I will keep that in mind